src/Compiler/CMakeLists.txt

```
@@ -74,11 +74,44 @@ add_onnx_mlir_library(OMCompilerDialects
   MLIRIR
   MLIROpenMPToLLVMIRTranslation
   ${dialect_libs}
-  )
+)
+
+add_onnx_mlir_library(OMDisposableGarbageCollector
+ DisposableGarbageCollector.cpp
+
+  EXCLUDE_FROM_OM_LIBS
+
+  INCLUDE_DIRS PRIVATE
+  ${FILE_GENERATE_DIR}
+
+  INCLUDE_DIRS PUBLIC
+  ${ONNX_MLIR_SRC_ROOT}/include
+
+  LINK_LIBS PUBLIC
+  OMONNXOps
+)
+
+add_onnx_mlir_library(OMOnnxToMlirPasses
+  OnnxToMlirPasses.cpp
+  
+  EXCLUDE_FROM_OM_LIBS
+
+  INCLUDE_DIRS PRIVATE
+  ${FILE_GENERATE_DIR}
+
+  INCLUDE_DIRS PUBLIC
+  ${ONNX_MLIR_SRC_ROOT}/include
+
+  LINK_LIBS PUBLIC
+  OMHybridTransform
+  OMONNXStandardFuncReturnPass
+  OMONNXSimplifyShapeRelatedOps
+  OMDisposableGarbageCollector
+  MLIRFuncDialect
```

NIT: I think these can be private, as they are not exposed in the header
```suggestion
  LINK_LIBS PRIVATE
  OMHybridTransform
  OMONNXStandardFuncReturnPass
  OMONNXSimplifyShapeRelatedOps
  OMDisposableGarbageCollector
  MLIRFuncDialect
```

---

done